src/components/organisms/NoteDetail.tsx

```diff
@@ -499,26 +635,49 @@ const Container = styled.div`
   flex-direction: column;
   height: 100%;
   & > .bottomBar {
+    ${backgroundColor};
     display: flex;
-    ${borderTop}
+    ${borderTop};
     height: 24px;
     & > :first-child {
       flex: 1;
     }
+    z-index: 5001;
+  }
+`
+
+/* todo: [komediruzecki-14/02/2021] It's either this width manipulation and
```

Comment:
Any better solution? Or is this fine?

Reply:
As I said above, we should put this in CodeEditor or put it in the same container of the editor at least

```
<div className='splitLeft'>
{this.state.showSearch && <div className='search'><LocalSearch/></div>}
<div className='editor'>{codeEditor}</div>
</div>
```

Reply:
This all seems perfectly fine in theory, but either I am very bad at CSS styling or it is really hard to put it this way with CodeMirror text area which takes 100% height (read this todo).

I tried today again, to put it like you shown in example, but the search then just don't show properly, even after configuring CSS a bit better (position relative etc.), I still cannot get it to move the codeEditor visually after (not on top) the search bar. As described in this todo, the height: 100% on code editor makes it hard to put the search bar component on top + sticky + move the code editor down when it expands (in such a way that the first line of the editor is seen).

Maybe we should try together, or if you have some advice on how to do this with code editor setup like this and add a magic CSS rule which will work in all cases :)